src/components/schemas/statements.yaml

```diff
@@ -0,0 +1,149 @@
+statements_in:
+  type: object
+  properties:
+    file_id:
+      allOf:
+        - $ref: './fields.yaml#/id_string'
+      example: fidftadfdsdf
+  required:
+    - file_id
+
+statements_out:
+  type: object
+  additionalProperties: False
+  properties:
+    id:
+      allOf:
+        - $ref: './fields.yaml#/id_string'
+      example: stmtwownwrng
+    org_id:
+      $ref: './fields.yaml#/org_id'
+    user_id:
+      $ref: './fields.yaml#/user_id'
+    created_at:
+      $ref: './fields.yaml#/created_at'
+    updated_at:
+      $ref: './fields.yaml#/updated_at'
+    file_name:
+      type: string
+      description: |
+        Specifies the name of file uploaded by the user.
+      example: amex.csv
+    file_size:
+      type: number
+      description: |
+        Specifies the size of file uploaded by the user. (The size is represented in bytes)
+      example: 21
+    no_of_debit:
+      type: integer
+      description: |
+        Number of debit transactions in the statement uploaded by the user.
+      example: 3
+    no_of_credit:
+      type: integer
+      description: |
+        Number of credit transactions in the statement uploaded by the user.
+      example: 3
+    no_of_transactions:
+      type: integer
+      description: |
+        Number of transactions in the statement uploaded by the user.
+      example: 6
+    statement_columns:
+      type: array
+      items: 
+        type: string
+      description: >
+        This specifies headers of the statement uploaded by the user.
+      example:  
+        - account_number
+        - amount
+    start_date:
+      $ref: './fields.yaml#/timestamptz_utc'
+      description: |
+        Start date of the statement period
+      nullable: true
+    end_date:
+      $ref: './fields.yaml#/timestamptz_utc'
+      description: |
+        End date of the statement period
+      nullable: true
+    has_statement_duplicate:
+      type: boolean
+      description: |
+        Indicates whether the statement is completely duplicate or not.
+    duplicate_statement_ids:
+      type: array
+      description:
+        Holds the list of statement_ids which are duplicate to the current statement
```

Comment:
Holds the list of statement_ids which are complete duplicate of the current statement